src/database/initialization/organization_to_json.sql

```diff
@@ -1,5 +1,5 @@
--- An explicit DROP is needed to remove the old function instead of overloading it.
-DROP FUNCTION IF EXISTS organization_to_json(organizations);
+SELECT drop_function('organization_to_json');
+
 CREATE OR REPLACE FUNCTION organization_to_json(organization organizations, authenticationId VARCHAR DEFAULT NULL)
```

Comment:
This is the only function we've changed the signature of, isn't it? What's the problem with this approach?

Reply:
We now have a piece of code that only exists because a signature, at on point in time, existed in the code base.

That's not a sustainable way to delete things (this isn't a migration).  Does that drop need to be there for the rest of time?  What happens the next time we change a function signature, do we just accumulate clutter in the file forever?

This function accomplishes what we had intended `CREATE OR REPLACE` to accomplish (but which doesn't)